fido-key-manager/src/main.rs

```diff
@@ -13,7 +13,7 @@ use std::time::Duration;
 use tokio_stream::StreamExt;
 #[cfg(feature = "solokey")]
 use webauthn_authenticator_rs::ctap2::SoloKeyAuthenticator;
-use webauthn_authenticator_rs::prelude::WebauthnCError;
+// use webauthn_authenticator_rs::prelude::WebauthnCError;
```

Comment:
This should be gated on `#[cfg(feature = "solokey"]` to fix the warning.

Reply:
K